crypto-first-war.md

```diff
@@ -0,0 +1,61 @@
+| Author | TallAmericano |
+| :---- | :---- |
+| Title | Crypto First War |
+| Headline  | Crypto First War examining the historical context, the rise of Cypherpunks, legislative battles, and the lasting impact on global encryption policies. |
+| Cypherpunktimes | cypherpunktimes.com/...link |
+| Publish Date | 00 Feb 2024 |
+
+| Social Media | Link |
+| :---- | :---- |
+| Twitter | post link… |
+| Twitter | thread link… |
+| Twitter | poll link… |
+| Reddit  | link… |
+| Bison Relay | News feed title |
+| Other platforms | Instagram and TikTok |
+
+Thesis statement - The importance of securing digital communications became increasingly apparent as computers became more popular. This article looks at the first crypto war, examining the historical context, the rise of Cypherpunks, legislative battles, and the lasting impact on global encryption policies.
```

Comment:
What the article is about is implied, no need to say things like "This article looks at". We're also not interested in the structure here. 

Thesis statement. Be to the point and be descriptive. What the article is about and your single most important point or argument.

For example:

"The innovation of computer technology gave birth to the information age. A period of unprecedented freedom of speech and knowledge sharing. Preserving these freedoms became the objectives of the first crypto war."

Reply:
What about : This article looks at the first crypto war, examining the historical context and the lasting impact on global encryption policies.

Reply:
This works for me 👍